src/Notification/DefaultNotification.php

```diff
@@ -121,8 +121,12 @@ public function setIconFromModel(string $icon, string $iconSize = null)
 		{
 			$iconSize = deserialize($iconSize);
 		}
-		$file = FilesModel::findByUuid($icon);
-		$image = System::getContainer()->get('contao.image.image_factory')->create($file->path, $iconSize);
-		$this->setIcon($image->getUrl(System::getContainer()->getParameter('kernel.project_dir')));
-	}
+                       
+        $rootDir = System::getContainer()->getParameter('kernel.project_dir');
+        $imageFactory = System::getContainer()->get('contao.image.image_factory');
+        if (($objModel = FilesModel::findByUuid($icon)) != null && is_file($rootDir . '/' . $objModel->path)) {
```

Comment:
Null check is not necassary, just test `if ($objModel = FilesModel::findByUuid($icon) && ...)` is enough

Reply:
Done